lib/loggerator/loggerator.rb

```diff
@@ -11,7 +11,7 @@ def log(data, &block)
     Log.to_stream(Log.stdout, Log.contexts(data), &block)
   end
 
-  def log_error(e=$!, data = {})
+  def log_error(e=$ERROR_INFO, data = {})
```

Comment:
@reidmix How would you approach fixing this? `def log_error(e, data)`, making the exception require?